src/tools/crawler/main.rs

```diff
@@ -75,10 +80,34 @@ fn start_logger(default_level: LevelFilter) {
 async fn main() {
     start_logger(LevelFilter::INFO);
     let args = Args::parse();
+    let mut seed_addrs: Vec<SocketAddr> = Vec::new();
 
     // Create the crawler with the given listener address.
     let crawler = Crawler::new().await;
 
+    if let Some(seeds) = args.seed_addrs {
+        seed_addrs.extend_from_slice(&seeds);
+    }
+
+    if let Some(dns_seed) = args.dns_seed {
+        for seed_str in dns_seed {
+            let response = lookup_host(&seed_str);
+
+            if let Ok(response) = response {
+                for address in response.iter() {
+                    seed_addrs.push(SocketAddr::new(*address, 8233)); // DNS addrs use this port
```

Comment:
maybe make this number a const

Reply:
Right, it's been already defined but wasn't public. 
Changed. 👍